Documentation/Blazorise.Docs/Pages/Docs/Components/Accordions/AccordionPage.razor

```diff
@@ -94,52 +94,4 @@
     </Alert>
 </Paragraph>
 
-<DocsPageSubtitle>
-    API
-</DocsPageSubtitle>
-
-<Heading Size="HeadingSize.Is3">
-    Attributes
-</Heading>
-
-<DocsAttributes Title="Accordion">
-    <DocsAttributesItem Name="ChildContent" Type="RenderFragment" Default="null">
-        Specifies the content to be rendered insidethe accordion.
-    </DocsAttributesItem>
-</DocsAttributes>
-
-<DocsAttributes Title="AccordionToggle">
-    <DocsAttributesItem Name="Clicked" Type="EventCallback<MouseEventArgs>" Default="null">
-        Occurs when the toggle button is clicked.
-    </DocsAttributesItem>
-    <DocsAttributesItem Name="ChildContent" Type="RenderFragment" Default="null">
-        Specifies the content to be rendered insidethe accordion toggle.
-    </DocsAttributesItem>
-</DocsAttributes>
-
-<DocsAttributes Title="AccordionItem">
-    <DocsAttributesItem Name="Visible" Type="bool" Default="false">
-        Defines the collapse state of the accordion item.
-    </DocsAttributesItem>
-    <DocsAttributesItem Name="VisibleChanged" Type="EventCallback<bool>">
-        Occurs when the accordion item visibility state changes.
-    </DocsAttributesItem>
-    <DocsAttributesItem Name="ChildContent" Type="RenderFragment" Default="null">
-        Specifies the content to be rendered inside the accordion item.
-    </DocsAttributesItem>
-</DocsAttributes>
-
-<DocsAttributes Title="AccordionHeader">
-    <DocsAttributesItem Name="Clicked" Type="EventCallback<MouseEventArgs>">
-        Occurs when the accordion header is clicked.
-    </DocsAttributesItem>
-    <DocsAttributesItem Name="ChildContent" Type="RenderFragment" Default="null">
-        Specifies the content to be rendered inside the accordion body.
-    </DocsAttributesItem>
-</DocsAttributes>
-
-<DocsAttributes Title="AccordionBody">
-    <DocsAttributesItem Name="ChildContent" Type="RenderFragment" Default="null">
-        Specifies the content to be rendered inside the accordion body.
-    </DocsAttributesItem>
-</DocsAttributes>
\ No newline at end of file
+<ComponentApiDocs ComponentTypes="[typeof(Accordion), typeof(AccordionToggle), typeof(AccordionItem), typeof(AccordionHeader), typeof(AccordionBody)]" />
```

Comment:
Could generics work somehow? That could be cool. Value1, Value2, Value3
i.e 
`<ComponentApiDocs Value1"Accordion" Value2="AccordionToggle" Value3="AccordionItem" Value4="AccordionHeader" Value5="AccordionBody" />`
Not sure if worth it though? Thoughts?
@stsrki @tesar-tech ?

Reply:
It doesn't make sense to me. We would need to have as many strongly-typed type params as there could be used at some point. Better to leave it as it is, with `typeof()`.